docs/provider-specific.md

```diff
@@ -118,3 +118,74 @@ services:
           max-size: 10m
       image: haugene/transmission-openvpn:latest
 ```
+
+### PROTONVPN
+
+[PROTONVPN](https://protonvpn.com/support/linux-openvpn/#preparation) provides `.ovpn` configuration files. Just download the one you want to connect with and which allows p2p.
+
+### Prerequisites:
+User needs to have a payed account.
+
+1. download your protonvpn ovpn file from a destination which allows p2p.
+2. in the directory with your docker-compose file, create a directory: `mkdir protonvpn`
+3. copy your ovpn file (node-<country of choice>.protonvpn.net.udp.ovpn) from step 1 to the protonvpn directory
+4. add the environment vars below and add +pmp to your username if you want to use port forwarding.
+5. add the [update-port.sh](https://github.com/haugene/vpn-configs-contrib/blob/main/openvpn/protonvpn/update-port.sh) script for protonvpn from vpn-configs-contrib to the protonvpn directory of step 2.
```

Comment:
```suggestion
5. add the [update-port.sh](https://github.com/haugene/vpn-configs-contrib/blob/main/openvpn/protonvpn/update-port.sh) script for protonvpn from vpn-configs-contrib to the ProtonVPN directory of step 2.
```

Reply:
The directory is named protonvpn in step2. So it wouldn't be logical to call the same directory in step 5 ProtonVPN?

Reply:
Whoops, my bad, fixed the wrong word. Edited above.